pkg/build/api/types.go

```diff
@@ -411,6 +411,8 @@ const (
 	// BuildConfigLabelDeprecated was used as BuildConfigLabel before adding namespaces.
 	// We keep it for backward compatibility.
 	BuildConfigLabelDeprecated = "buildconfig"
+	// BuildConfigDeletionAnnotation is an annotation that marks a BuildConfig that is being deleted
+	BuildConfigDeletionAnnotation = "openshift.io/build-config.deletion-pending"
```

Comment:
> We've talked about making deployments inert to keep them from doing work, this looks parallel to that.

@ironcladlou @deads2k does this mean that the dc controller should ignore altogether a deployment (that is explicitly marked as inert via a spec field) or the lack of a config change trigger is enough for the deployment to be considered inert?